src/com/amazon/corretto/crypto/provider/LibCryptoRng.java

```diff
@@ -10,7 +10,7 @@
  * A simple wrapper around the linked LibCrypto's RAND_bytes() API.
  */
 public class LibCryptoRng extends SecureRandom {
-    private static final String ALGORITHM_NAME = "LibCryptoRng";
+    public static final String ALGORITHM_NAME = "LibCryptoRng";
```

Comment:
Any reason we can't use the public `getAlgorithm()` mathod? Or do we need the static accessor here?

Reply:
`getAlgorithm()` is not static, as you pointed out.